DuckDuckGo/TutorialSettings.swift

```diff
@@ -28,13 +29,23 @@ struct OnboardingSettings {
         }
     }
     
-    public var instructionsFirstLaunch: Bool {
+    public var hasSeenSafariSearchInstructions: Bool {
         get {
-            guard let userDefaults = userDefaults() else { return true }
-            return userDefaults.bool(forKey: Keys.instructionsFirstLaunch, defaultValue: true)
+            guard let userDefaults = userDefaults() else { return false }
```

Comment:
I think this should be `return true`. You let me know that this could never happen (like approaching 0 possibility).